docs/fieldnorm/control_of_documents.md

```diff
@@ -1,25 +1,91 @@
 # Control of documents
 
-Bioinformaticians work in a git environment. Git is a distributed version control system that tracks changes in any set of computer files, usually used for coordination work among programmers collaboratively developing source code during software development. Software procedures and related documentation needs to be controlled, and according to ISO-15189, this can be in any form or type of medium. For software development, there are several reasons to not include all the software documentation within the medical laboratories chosen quality management system.
+## Why do Bioinformaticians need a different QMS
 
-- Software development is done in a distributed version control system. These systems are used for coordination work among programmers. Often these systems also allow for documentation of procedures. With the following advantages a. Automatically link to i.e. code, issues, releases, and release notes. b. Possibility to include code blocks that allow for copy-paste into the console/terminal. It’s fast and readable for bioinformaticians. For example, code blocks can be used for general procedures to start a pipeline, or for trouble shooting.
-- Software development, and hence, corresponding documentation is prone to frequent updates. With the QMS of the laboratory, there are often waiting steps for different members of the laboratory to verify or authorize documentation. Hence, there might be a delay in release of a new version as the documentation of the previous release is not authorized yet.
-- add more
+_Bioinformaticians_ work in a fairly different environement than biologists, and the classic Quality Management System (QMS) offered to biologists are often impractical for _Bioinformaticians_ needs. Here is a list of the most wanted features :
 
-To allow for a git-like QMS to be used by bioinformaticians the following aspects need to be considered, and documented within the QMS how compliance is achieved. ISO-15189:2022, 8.3.1
+- Code blocks can be copy/pasted safely, as opposed to standard QMS with a word or pdf format, which introduce erroneous changes on some special characters (such as "-")
+- Linking versions of the _documentation_ with the corresponding release number of the documented tool
```

Comment:
hyperlinking?? (linking can be done on paper as well)   - and keeping these links up to date (???)

Reply:
Here the sentence from below: As the workflow often get upgraded and updated, their documentation, as well, is susceptible to change. Which is why many chose _Bioinformaticians_ their own form of QMS. Examples of those documentations systems are: --> but it's kind of already here

Reply:
I am not sure I understand the suggestion here sorry :x Can you explain again ?